src/components/react-hook-form/utils/cancel-button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+const CancelButton = ({ onClick, variant, disabled }) => {
+    return (
+        <Button
+            onClick={onClick}
+            variant={variant}
+            disabled={disabled}
+            sx={{ color: 'inherit' }}
```

Comment:
Why don't you use the color of the theme as suggested in the US ? How can you be sure inherit give you the color you want ?

Reply:
What happen if you remove the inherit ?

Reply:
If I remove "inherit", all the buttons take on the default MUI color which is blue in gridexplore. That's the reason why I placed this component in commons UI, so I don't have to repeat "inherit" and it's only to cancel it out.